langchain4j-google-ai-gemini/src/test/java/dev/langchain4j/model/googleai/GoogleAiGeminiChatModelIT.java

```diff
@@ -510,6 +511,42 @@ void should_handle_enum_type() {
             .isEqualTo("{\"sentiment\": \"POSITIVE\"}");
     }
 
+    @Test
+    void should_handle_enum_output_mode() {
+        // given
+        GoogleAiGeminiChatModel gemini = GoogleAiGeminiChatModel.builder()
+            .apiKey(GOOGLE_AI_GEMINI_API_KEY)
+            .modelName("gemini-1.5-flash")
+            .logRequestsAndResponses(true)
+            .responseMimeType("text/x.enum")
+            .responseFormat(ResponseFormat.builder()
+                .type(ENUM)
+                .jsonSchema(JsonSchema.builder()
```

Comment:
Somehow this feels wrong... type is `ENUM` but `JSON` schema is specified.
What about using `JSON` type (remove `ENUM` from `ResponseFormatType`) and map to `text/x.enum` when `rootElement` is of type `JsonEnumSchema`?
I doubt many people are aware of `text/x.enum`. And I bet most people associate "structured outputs" with JSON

Reply:
I removed the new `ENUM` value from `ResponseFormatType` that I had added.
You're correct that `text/x.enum` is a bit mysterious, can be hidden, and after all, it's just a flavor of `JSON`.